ruby/generate-map/index.rb

```diff
@@ -0,0 +1,59 @@
+require 'open-street-map'
```

Comment:
I seem to get an error running this code:

```shell
$ curl -H "X-Internal-Challenge: secret-key" \
>     -H "Content-Type: application/json" \
>     -X POST http://localhost:3000/ \
>     -d '{"payload": "{}"}' | jq
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  4723  100  4706  100    17   135k    500 --:--:-- --:--:-- --:--:--  135k
{
  "code": 500,
  "message": "/usr/code-start/index.rb:1:in `require'\n/usr/code-start/index.rb:1:in `<top (required)>'\n/usr/local/src/server.rb:74:in `load'\n/usr/local/src/server.rb:74:in `block in <top (required)>'\n/usr/local/src/vendor/bundle/ruby/3.1.0/gems/sinatra-3.0.2/lib/sinatra/base.rb:1684:in `call'\n/usr/local/src/vendor/bundle/ruby/3.1.0/gems/sinatra-3.0.2/lib/sinatra/base.rb:1684:in `block in compile!'\n/usr/local/src/vendor/bundle/ruby/3.1.0/gems/sinatra-3.0.2/lib/sinatra/base.rb:1002:in `block (3 levels) in route!'\n/usr/local/src/vendor/bundle/ruby/3.1.0/gems/sinatra-3.0.2/lib/sinatra/base.rb:1020:in `route_eval'\n/usr/local/src/vendor/bundle/ruby/3.1.0/gems/sinatra-3.0.2/lib/sinatra/base.rb:1002:in `block (2 levels) in route!'\n/usr/local/src/vendor/bundle/ruby/3.1.0/gems/sinatra-3.0.2/lib/sinatra/base.rb:1051:in `block in process_route'\n/usr/local/src/vendor/bundle/ruby/3.1.0/gems/sinatra-3.0.2/lib/sinatra/base.rb:1049:in `catch'\n/usr/local/src/vendor/bundle/ruby/3.1.0/gems/sinatra-3.0.2/lib/sinatra/base.rb:1049:in `process_route'\n/usr/local/src/vendor/bundle/ruby/3.1.0/gems/sinatra-3.0.2/lib/sinatra/base.rb:1000:in `block in route!'\n/usr/local/src/vendor/bundle/ruby/3.1.0/gems/sinatra-3.0.2/lib/sinatra/base.rb:997:in `each'\n/usr/local/src/vendor/bundle/ruby/3.1.0/gems/sinatra-3.0.2/lib/sinatra/base.rb:997:in `route!'\n/usr/local/src/vendor/bundle/ruby/3.1.0/gems/sinatra-3.0.2/lib/sinatra/base.rb:1121:in `block in dispatch!'\n/usr/local/src/vendor/bundle/ruby/3.1.0/gems/sinatra-3.0.2/lib/sinatra/base.rb:1092:in `catch'\n/usr/local/src/vendor/bundle/ruby/3.1.0/gems/sinatra-3.0.2/lib/sinatra/base.rb:1092:in `invoke'\n/usr/local/src/vendor/bundle/ruby/3.1.0/gems/sinatra-3.0.2/lib/sinatra/base.rb:1116:in `dispatch!'\n/usr/local/src/vendor/bundle/ruby/3.1.0/gems/sinatra-3.0.2/lib/sinatra/base.rb:932:in `block in call!'\n/usr/local/src/vendor/bundle/ruby/3.1.0/gems/sinatra-3.0.2/lib/sinatra/base.rb:1092:in `catch'\n/usr/local/src/vendor/bundle/ruby/3.1.0/gems/sinatra-3.0.2/lib/sinatra/base.rb:1092:in `invoke'\n/usr/local/src/vendor/bundle/ruby/3.1.0/gems/sinatra-3.0.2/lib/sinatra/base.rb:932:in `call!'\n/usr/local/src/vendor/bundle/ruby/3.1.0/gems/sinatra-3.0.2/lib/sinatra/base.rb:921:in `call'\n/usr/local/src/vendor/bundle/ruby/3.1.0/gems/rack-protection-3.0.2/lib/rack/protection/xss_header.rb:20:in `call'\n/usr/local/src/vendor/bundle/ruby/3.1.0/gems/rack-protection-3.0.2/lib/rack/protection/path_traversal.rb:18:in `call'\n/usr/local/src/vendor/bundle/ruby/3.1.0/gems/rack-protection-3.0.2/lib/rack/protection/json_csrf.rb:28:in `call'\n/usr/local/src/vendor/bundle/ruby/3.1.0/gems/rack-protection-3.0.2/lib/rack/protection/base.rb:53:in `call'\n/usr/local/src/vendor/bundle/ruby/3.1.0/gems/rack-protection-3.0.2/lib/rack/protection/base.rb:53:in `call'\n/usr/local/src/vendor/bundle/ruby/3.1.0/gems/rack-protection-3.0.2/lib/rack/protection/frame_options.rb:33:in `call'\n/usr/local/src/vendor/bundle/ruby/3.1.0/gems/rack-2.2.4/lib/rack/logger.rb:17:in `call'\n/usr/local/src/vendor/bundle/ruby/3.1.0/gems/rack-2.2.4/lib/rack/common_logger.rb:38:in `call'\n/usr/local/src/vendor/bundle/ruby/3.1.0/gems/sinatra-3.0.2/lib/sinatra/base.rb:261:in `call'\n/usr/local/src/vendor/bundle/ruby/3.1.0/gems/sinatra-3.0.2/lib/sinatra/base.rb:254:in `call'\n/usr/local/src/vendor/bundle/ruby/3.1.0/gems/rack-2.2.4/lib/rack/head.rb:12:in `call'\n/usr/local/src/vendor/bundle/ruby/3.1.0/gems/rack-2.2.4/lib/rack/method_override.rb:24:in `call'\n/usr/local/src/vendor/bundle/ruby/3.1.0/gems/sinatra-3.0.2/lib/sinatra/base.rb:219:in `call'\n/usr/local/src/vendor/bundle/ruby/3.1.0/gems/sinatra-3.0.2/lib/sinatra/base.rb:1995:in `call'\n/usr/local/src/vendor/bundle/ruby/3.1.0/gems/sinatra-3.0.2/lib/sinatra/base.rb:1554:in `block in call'\n/usr/local/src/vendor/bundle/ruby/3.1.0/gems/sinatra-3.0.2/lib/sinatra/base.rb:1770:in `synchronize'\n/usr/local/src/vendor/bundle/ruby/3.1.0/gems/sinatra-3.0.2/lib/sinatra/base.rb:1554:in `call'\n/usr/local/src/vendor/bundle/ruby/3.1.0/gems/puma-6.0.0/lib/puma/configuration.rb:268:in `call'\n/usr/local/src/vendor/bundle/ruby/3.1.0/gems/puma-6.0.0/lib/puma/request.rb:89:in `block in handle_request'\n/usr/local/src/vendor/bundle/ruby/3.1.0/gems/puma-6.0.0/lib/puma/thread_pool.rb:343:in `with_force_shutdown'\n/usr/local/src/vendor/bundle/ruby/3.1.0/gems/puma-6.0.0/lib/puma/request.rb:88:in `handle_request'\n/usr/local/src/vendor/bundle/ruby/3.1.0/gems/puma-6.0.0/lib/puma/server.rb:430:in `process_client'\n/usr/local/src/vendor/bundle/ruby/3.1.0/gems/puma-6.0.0/lib/puma/server.rb:233:in `block in run'\n/usr/local/src/vendor/bundle/ruby/3.1.0/gems/puma-6.0.0/lib/puma/thread_pool.rb:150:in `block in spawn_thread'"
}
```

in the runtime logs, i see:

```
#<LoadError: cannot load such file -- open-street-map>
```

Are you having this problem?